### PR TITLE
Add positive invariant test for ESSL3 shader

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

File: sdk/tests/conformance2/glsl3/valid-invariant.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+invariant v_varying;
+
+void main()
+{
+    v_varying = vec4(0.0, 0.0, 0.0, 1.0);
+    gl_Position = v_varying;
+}
+</script>
+<script id="vertexShaderGlobalInvariant" type="text/something-not-javascript">#version 300 es
+#pragma STDGL invariant(all)
```
Comment (zhenyao) [inline sdk/tests/conformance2/glsl3/valid-invariant.html]: I vaguely remember this is no longer valid in ESSL3? I could be wrong though.

File: sdk/tests/conformance2/glsl3/00_test_list.txt [hunk 2]
```diff
@@ -34,5 +34,6 @@ texture-offset-out-of-range.html
 --min-version 2.0.1 unary-minus-operator-in-dynamic-loop.html
 uniform-block-layouts.html
 uniform-location-length-limits.html
+valid-invariant.html
```
Comment (zhenyao) [inline sdk/tests/conformance2/glsl3/00_test_list.txt]: Is this test currently passing?  If not, please put it to 2.0.1

Comment (qkmiao) [inline sdk/tests/conformance2/glsl3/00_test_list.txt]: See the patch I mentioned, https://chromium-review.googlesource.com/#/c/409157/

**Comment by zhenyao** [comment]
LGTM
This indeed catches a bug I recently introduced to ANGLE shader translator.  Let's land it for 2.0.0.

**Pull request merged**